py-polars/src/series/aggregation.rs

```
@@ -6,6 +6,20 @@ use crate::PySeries;
 
 #[pymethods]
 impl PySeries {
+    fn any(&self) -> Option<bool> {
+        match self.series.dtype() {
+            DataType::Boolean => Some(self.series.bool().unwrap().any()),
+            _ => None,
+        }
+    }
+
+    fn all(&self) -> Option<bool> {
+        match self.series.dtype() {
+            DataType::Boolean => Some(self.series.bool().unwrap().all()),
+            _ => None,
```

This should raise an error informing that we expect a bool here.